doc/ot_api_doc.h

```diff
@@ -205,5 +205,15 @@
  * @defgroup plat-toolchain              Toolchain
  * @defgroup plat-trel                   TREL - Platform
  *
+ * @defgroup plat-provisional         Provisional
```

Comment:
Add more spaces to align the titles with those above?  Did you run make-pretty already?